src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/MLKemCertTests.cs

```diff
@@ -0,0 +1,137 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Security.Cryptography.Tests;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    public static class MLKemCertTests
+    {
+        public static bool MLKemIsNotSupported => !MLKem.IsSupported;
+
+        [ConditionalFact(typeof(MLKem), nameof(MLKem.IsSupported))]
```

Comment:
Ah yeah, that actually makes the test better.

```suggestion
        [Fact]
```